CS: clean up after merges

jrfnl opened the pull request:
## Summary
This PR can be summarized in the following changelog entry:
_N/A_

## Relevant technical choices:
* No functional changes.
* Code style compliance.

Mostly whitespace fixes, a missing comma after last array item and a missing semi-colon in embedded PHP.


## Test instructions
_N/A_

Xyfi commented:
Acceptance :construction: 

Some code style issues still need to be fixed. Also the sniffs don't seem to make sense.

<img width="517" alt="screen shot 2018-08-08 at 13 26 58" src="https://user-images.githubusercontent.com/5389513/43834624-04a58794-9b0f-11e8-97f7-8ff35da0b09d.png">

jrfnl commented:
@Xyfi He Alexander, this has to do with the antiquated version of PHPCS which is still used in this plugin.
For now, I have removed those three changes (commit amended), the build should now pass without problem. These three changes will be addressed in another PR later on.

On another: please ping me to rebase PRs if needed instead of merging trunk into them. For this PR, I've now rebased it anyway and force pushed it to get rid of the trunk merge.
